spec/polaris-management-service.yml

```diff
@@ -887,6 +889,16 @@ components:
             this is specified as the 'warehouse' when multiple logical catalogs are served under the same base
             uri, and often translates into a 'prefix' added to all REST resource paths
 
+    HadoopConnectionConfigInfo:
+      type: object
+      description: Configuration necessary for connecting to a Hadoop Catalog
+      allOf:
+        - $ref: '#/components/schemas/ConnectionConfigInfo'
+      properties:
+        warehouse:
```

Comment:
Just want to flag that, for `IcebergRestConnectionConfigInfo`, we use `remoteCatalogName`, and it's passed as the warehouse property when initializing the catalog client. Using `warehouse` make more sense for `HadoopConnectionConfigInfo`.